Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core/Common/BussinessRules/BusinessErrors.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.Contracts.Core.Common.BussinessRules;
+
+public static class BusinessRuleError
+{
+    public const int Type = 10;
```

Comment:
Assign 100 here - there is a chance that 10 will be reached in one of the next versions easily - it is 6 today.

Btw. Maybe it would make sense to create ErrorOr custom enumerator and then here it will be ErrorOrCustomType = ErrorOrCustomType.HereTheName. Wdyt?

Reply:
Can you move it to a package that we have for building blocks in chapter 4?

Reply:
Yes